packages/dashboard-frontend/src/task-list/components/task-row.js

```diff
@@ -53,32 +54,63 @@ const LoadingTaskRow = ( { title } ) => {
  */
 export const TaskRow = ( { title, duration, priority, badge, isCompleted, onClick, children } ) => {
 	const svgAriaProps = useSvgAria();
+	const [ isButtonFocused, , ,handleButtonFocus, handleButtonBlur ] = useToggleState( false );
 
-	return <Table.Row>
-		<Table.Cell className="yst-font-medium yst-text-slate-800">
-			<div className="yst-flex yst-items-center yst-gap-2">
-				{ isCompleted
-					? <CheckCircleIcon className="yst-w-4 yst-text-green-500" { ...svgAriaProps } />
-					: <Ellipse className="yst-w-4 yst-text-slate-200" { ...svgAriaProps } /> }
-				{ title }
-				{ badgeOptions.includes( badge ) && <TaskBadge type={ badge } /> }
-			</div>
-		</Table.Cell>
-		<Table.Cell>
-			<Duration minutes={ duration } />
-		</Table.Cell>
-		<Table.Cell>
-			<Priority level={ priority } />
-		</Table.Cell>
-		<Table.Cell className="yst-align-middle">
-			<div className="yst-flex yst-items-center yst-justify-around yst-gap-2">
-				<button onClick={ onClick } aria-label={ __( "Open task modal", "wordpress-seo" ) }>
-					<ChevronRightIcon className="yst-w-4 yst-text-slate-800 rtl:yst-rotate-180" { ...svgAriaProps } />
-				</button>
-			</div>
-			{ children }
-		</Table.Cell>
-	</Table.Row>;
+	// Row hover or button focus triggers hover styles
+	const rowClass = classNames(
+		"yst-cursor-pointer yst-group",
+		{ "yst-bg-slate-50": isButtonFocused }
+	);
+
+	return (
+		<Table.Row className={ rowClass } onClick={ onClick } aria-label={ __( "Open task modal", "wordpress-seo" ) }>
+			<Table.Cell className={ classNames( "group-hover:yst-bg-slate-50", { "yst-bg-slate-50": isButtonFocused } ) }>
```

Comment:
The `classNames( "group-hover:yst-bg-slate-50", { "yst-bg-slate-50": isButtonFocused } )` part is true for all three table cells, maybe we can avoid code duplication?